Modules/Office365DSC/Office365DSCRequiredModules.ps1

```diff
@@ -0,0 +1,5 @@
+Import-Module -Name 'Microsoft.Online.SharePoint.PowerShell' -Version '16.0.8316.0' -Force -Verbose:$false -Global
```

Comment:
Not sure what the advantage of this approach would be. By using RequiredModules instead, it forces PowerShellGet to build the dependencies tree and automatically download all dependencies from the gallery.

Reply:
Trying to avoid verbose messages from module imports from showing when running `Start-DscConfiguration -Verbose`

Reply:
I opened a UserVoice request for this: https://windowsserver.uservoice.com/forums/301869-powershell/suggestions/37075921-requiredmodules-statement-in-module-manifests-shou